src/api.ts

```diff
@@ -28,6 +28,7 @@ import getAccountInfo from './ledger/accountinfo'
 import getAccountObjects from './ledger/accountobjects'
 import getPaymentChannel from './ledger/payment-channel'
 import preparePayment from './transaction/payment'
+import createPaymentTransaction from './transaction/payment'
```

Comment:
This uses the default export. You'll need to use something like `{createPaymentTransaction}`